components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -313,7 +314,9 @@ export class Panel extends React.Component<Props, State> {
               donationAction={this.showDonateToSiteDetail}
               onAmountChange={this.doNothing}
               onIncludeInAuto={this.switchAutoContribute}
-            />
+              />
+              <div style={{ height: '48px' }}></div>
```

Comment:
@ryanml This feels pretty hacky and may lead to confusion / bugs down the road. Does the parent have a fixed height? Can't we specify a min-height on this item then? or `100%`, or `height: -webkit-fill-available`

Reply:
@petemill this is definitely meant to be temporary (so it can land in `0.58`, until we work the ToggleTips component in to the Rewards Panel, this will all be handled in `brave-ui`.

I can look in to implementing this patch in a different way

Reply:
data attribute added to denote temporary status